### ISSUE TITLE
Struggling with WRF Aggregation

### ISSUE THREAD

**Issue by rsignell-usgs** [nctoolbox/nctoolbox#30]
Rutgers has been struggling for days with NCTOOLBOX access to aggregated WRF data.  

The issue is that `grid_interop` is always supposed to return `time lon lat` but in this case, it's returning the original variable names, and also has a singleton dimension:

```
 url = 'http://tds.maracoos.org/thredds/dodsC/RUWRF-3km-ROMS-Agg.nc';
 nc = ncgeodataset(url);
 v = nc.geovariable('T2');
 data = v.data(1,:,:);
 geo = v.grid_interop(1,:,:);
```

returns

```
>> geo

geo = 

    XLONG: [1x324x324 single]
     XLAT: [1x324x324 single]
     time: 735584
```

WRF is kind of strange in that it really does have lon,lat as functions of time:

```
>> size(nc.variable('XLONG'))

ans =

        1852         324         324
```

 Is that the problem?

Thanks,
Rich

**Comment by dblodgett-usgs** [comment]
Can't tell you how many times I've googled for a problem I have only to have @rsignell-usgs answer it. This whole conventions things and how the CDM does IOSP picking can be infuriating... Also, is WRF actually a convention?

**Comment by rsignell-usgs** [comment]
Agreed.   WRF is not really a convention -- NJ has a custom reader for it because it's very common in the community Unidata supports.
